installer/installer.py

```diff
@@ -542,10 +544,14 @@ def setup_cosmovisor(self):
             self.remove_safe("CHANGELOG.md")
             self.remove_safe("README.md")
             self.remove_safe("LICENSE")
-            self.mkdir_p(self.cosmovisor_root_dir)
-            self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis"))
-            self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis/bin"))
-            self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "upgrades"))
+            
+            if not is_cosmovisor_bump_needed:
+                self.mkdir_p(self.cosmovisor_root_dir)
+                self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis"))
+                self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "genesis/bin"))
+                self.mkdir_p(os.path.join(self.cosmovisor_root_dir, "upgrades"))
+            else:
+                self.log(f"Bumping Comsovisor binary to {DEFAULT_LATEST_COSMOVISOR_VERSION}")
```

Comment:
As far as I understand, this will only log the message and do absolutely nothing else (which is incorrect)